l10n_nl_dutch_company_type/data/l10n_nl_dutch_company_types.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You have the same file twice, one with plural one with singular name. Use git rm